task_test.go

```diff
@@ -1216,6 +1216,44 @@ func TestIncludesInternal(t *testing.T) {
 	}
 }
 
+func TestIncludesFlatten(t *testing.T) {
+	const dir = "testdata/includes_flatten"
+	tests := []struct {
+		name           string
+		taskfile       string
+		task           string
+		expectedErr    bool
+		expectedOutput string
+	}{
+		{"included flatten", "Taskfile.yml", "gen", false, "gen from included\n"},
```

Comment:
Please use fields name for this struct.

It could lead to errors otherwise, fields can be reordered, value could be assigned yo the wrong field